### PR TITLE
add Dockerfile

### PR DESCRIPTION
**Pull request by TMHBOFH**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Pulling Ubuntu image
+FROM ubuntu:22.04
```
Comment (notheowner) [inline Dockerfile]: I don't know about an entire Ubuntu image.
Alpine maybe?

Comment (TMHBOFH) [reply Dockerfile]: I haven't done much with Alpine, but I like to watch it

Comment (TMHBOFH) [reply Dockerfile]: Hi, i has switch to Alpine based image

**Comment by TMHBOFH** [comment]
not needed

**Comment by skelgaard** [comment]
why was this dropped ?
There is several good reasons for using docker.
- You will isolate the code from others in a docker and only give it access to what you want instead of your whole system.
- Its alot eaiser to update and you newer have any kind of conflicts with others, as its all isolated in own enviroment.
- its also easier to move to another server, if you need to, as its only a matter of moving the shared folders/files.

**Pull request closed without merging**